apps/builder/app/builder/features/props-panel/props-panel.tsx

```diff
@@ -12,24 +11,55 @@ import {
   ComboboxListboxItem,
   TextField,
   SmallIconButton,
+  Separator,
+  Flex,
+  Text,
+  CSS,
 } from "@webstudio-is/design-system";
 import { ChevronDownIcon, PlusIcon } from "@webstudio-is/icons";
 import type { Publish } from "~/shared/pubsub";
 import { propsStore, useInstanceProps } from "~/shared/nano-states";
-import { CollapsibleSection, ComponentInfo } from "~/builder/shared/inspector";
+import { CollapsibleSection } from "~/builder/shared/inspector";
 import {
   useStyleData,
   type SetProperty as SetCssProperty,
 } from "~/builder/features/style-panel/shared/use-style-data";
 import { renderControl } from "./controls/combined";
 import { usePropsLogic, type NameAndLabel } from "./use-props-logic";
 import { type PropMeta, type PropValue, getLabel } from "./shared";
-
 import { useState } from "react";
+import {
+  getComponentPropsMeta,
+  getComponentMeta,
+  type WsComponentMeta,
+} from "@webstudio-is/react-sdk";
 
 const itemToString = (item: NameAndLabel | null) =>
   item ? getLabel(item, item.name) : "";
 
+const Row = ({ children, css }: { children: React.ReactNode; css?: CSS }) => (
+  <Flex css={{ px: theme.spacing[9], ...css }} gap="2" direction="column">
+    {children}
+  </Flex>
+);
+
+const InstanceInfo = ({
+  componentMeta,
```

Comment:
will have a small conflict with my pr https://github.com/webstudio-is/webstudio-builder/pull/1146